src/app/services/videoprocessingcuda/src/stream/input/IVideoInput.h

```diff
@@ -0,0 +1,15 @@
+#ifndef I_VIDEO_INPUT_H
+#define I_VIDEO_INPUT_H
+
+#include "utils/images/Images.h"
+
+class IVideoInput
+{
+public:
+
+    virtual ~IVideoInput() {};

```

Comment:
I prefer to use `= default` since it silence a clang warning

Reply:
I will change it in all interfaces, I forgot about that